sdk/hybridnetwork/Azure.ResourceManager.HybridNetwork/samples/Generated/Samples/Sample_ConfigurationGroupSchemaResource.cs

```diff
@@ -118,7 +118,7 @@ public async Task Update_CreateOrUpdateTheConfigurationGroupSchemaResource()
         public async Task UpdateState_UpdateNetworkServiceDesignVersionState()
         {
             // Generated from example definition: specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/stable/2023-09-01/examples/ConfigurationGroupSchemaVersionUpdateState.json
-            // this example is just showing the usage of "ConfigurationGroupSchemas_UpdateState" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "ConfigurationGroupSchemas_updateState" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `updateSate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/hybridnetwork/resource-manager/Microsoft.HybridNetwork/stable/2024-04-15/configurationGroupSchema.json#L299